ui/app/components/auth/fields.ts

```diff
@@ -20,15 +18,4 @@ interface Field {
 export default class AuthFields extends Component<Args> {
   // token or password should render as "password" types, otherwise render text inputs
   setInputType = (field: string) => (['token', 'password'].includes(field) ? 'password' : 'text');
```

Comment:
This is small enough it could happen in the component, but I thought the interfaces might be helpful in the future: 
https://github.com/hashicorp/vault/blob/e95f1455cc2b79a9cdc323aaa0c4f6099fb41102/ui/app/components/auth/fields.ts#L8-L16

If they seem unnecessary, happy to remove the backing class!